Domain/sdLitica.Events/Extensions/EventExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+using sdLitica.Events.Abstractions;
+using sdLitica.Messages.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace sdLitica.Events.Extensions
+{
+    public static class EventExtensions
+    {
+        public static IMessage ToMessage(this IEvent @event)
```

Comment:
Ok, for IEvent now I see why you need it. So later we will have something like
IEvent -> Event -> AnalyticsRequest and other messages we are going to send across the system will be inherited like this

Reply:
Yes, thats it. Please, see my explanation about Trace and Log in the other comment. Thanks.